packages/@dcl/ecs/src/components/extended/InputModifier.ts

```diff
@@ -0,0 +1,59 @@
+import { IEngine, LastWriteWinElementSetComponentDefinition, Entity } from '../../engine'
+import { InputModifier, PBInputModifier_StandardInput, PBInputModifier } from '../generated/index.gen'
+import {} from '../generated/InputModifier.gen'
+
+/**
+ * @public
+ */
+export interface InputModifierHelper {
+  /**
+   * @returns a input modifier mode
+   */
+  Standard: (standard: PBInputModifier_StandardInput) => PBInputModifier['mode']
+}
+
+/**
+ * @public
+ */
+export interface InputModifierComponentDefinitionExtended
+  extends LastWriteWinElementSetComponentDefinition<PBInputModifier> {
+  /**
+   * InputModifier helper with constructor
+   */
+  InputModifier: InputModifierHelper
+
+  /**
+   * Create or replace the component InputModifier in the entity specified
+   * @param entity - the entity to link the component
+   * @param inputModifier - the data for this input modifier
+   */
+  addStandardModifier: (entity: Entity, inputModifier: PBInputModifier_StandardInput) => void
```

Comment:
do we need this helper ? It seems that the Mode.Standard could do the trick and avoid generating extra methods